src/forms/helpers/useFormControl.js

```diff
@@ -3,11 +3,15 @@ import { isEmptyStringLike, isBoolean } from 'js-var-type';
 import { FormContext } from './form-helpers';
 import { toDatetimeLocal, fromDatetimeLocal } from '../../utils/formatters';
 
-export function useFormControl(name, type) {
+export function useFormControl(name, type, afterChange) {
```

Comment:
Rename the argument to "hooks" that will be an object witch will receive the `afterChange`